controllers/topics.controllers.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+
+
+const topicData = require('../db/data/development-data/topics');
```

Comment:
Why are you requiring in the data from the dev-data folder? What if you didn't have access to this information?